src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -371,7 +373,7 @@ public void handle(Endpoint<?> endpoint, Handler<?> handler) {
         try {
             bodyAsJson = routingContext.getBodyAsJson();
         } catch (DecodeException ex) {
-            
+
```

Comment:
If you're going to ignore an exception you should at least add a comment about why that is the correct thing to do.

Reply:
It seems I just removed some white spaces there. The code is from @ppatierno

Reply:
The comment is just before the `try`

Reply:
How do you know that the body is empty (and thus that it's OK to ignore the exception, as you say)? Shouldn't the client not send a body, rather than sending an empty one?

Reply:
I noticed that long time ago I guess that by empty I meant no body at all.